helm/api-testing/Chart.lock

```diff
@@ -1,6 +1,6 @@
 dependencies:
 - name: mongodb
   repository: oci://registry-1.docker.io/bitnamicharts
-  version: 15.1.3
-digest: sha256:40d43cd158585ff78c94e3eb2f46d2410488923a1e9242bf5c8c8af5bac0707d
-generated: "2024-04-09T11:17:10.1150205+08:00"
+  version: 15.1.7
+digest: sha256:e573c76d3c75c7e5bc61cbf1e7de46bbf59837de68ac06ebdfe1fa32337a6f58
+generated: "2024-05-02T19:54:38.5457449+08:00"
```

Comment:
Why upgrade it?

Reply:
this is what i generate things when run `make helm`, I will restore it.